ui/raidboss/data/04-sb/raid/o8n.js

```diff
@@ -227,16 +189,7 @@
       regexCn: Regexes.startsUsing({ id: '2916', source: '凯夫卡', capture: false }),
       regexKo: Regexes.startsUsing({ id: '2916', source: '케프카', capture: false }),
       suppressSeconds: 1,
-      infoText: {
-        en: 'Fake Ice: Get out',
-        fr: 'Fausse glace : Sortez',
-        de: 'Falsches Eis: Rausgehen',
-      },
-      tts: {
-        en: 'Get out',
-        fr: 'Sortez',
-        de: 'Rausgehen',
-      },
+      response: Responses.getOut(),
```

Comment:
I think the fake ice / true ice / etc is good for calling out here.  Same for all the other o8 ones.

Reply:
ok i reverted the changes